src/FixedDataTableCellDefault.js

```diff
@@ -70,44 +72,12 @@ class FixedDataTableCellDefault extends React.Component {
 
   render() {
     //Remove some props which we don't pass into div
-    var {
-      height,
-      width,
-      style,
-      className,
-      children,
-      columnKey,
-      columnIndex,
-      rowIndex,
-      left,
-      cellGroupType,
-      isHeader,
-      isGroupHeader,
-      maxWidth,
-      minWidth,
-      ...props
-    } = this.props;
 
-    var innerStyle = {
-      height,
-      width,
-      ...style,
-    };
+    const CellDefaultComponent = this.props.shouldUseLegacyComponents
+      ? CellDefaultLegacy
+      : CellDefault;
 
-    return (
-      <div
-        {...props}
-        className={joinClasses(
-          cx('fixedDataTableCellLayout/wrap'),
-          cx('public/fixedDataTableCell/wrap'),
-          cx('public/fixedDataTableCell/cellContent'),
-          className
-        )}
-        style={innerStyle}
-      >
-        {children}
-      </div>
-    );
+    return <CellDefaultComponent {...this.props} />;
```

Comment:
FixedDataTableCellDefault, aka DataCell, is essentially a type of cell that clients can utilize as a cell renderer. Clients have the flexibility to use any valid element or function as a cell renderer, and we have no control over the styles they apply to them.

we want to solely apply the necessary styles to the `content` within the FixedDataTableCell. This `content` is  what the client passes, which can be either a DataCell or any other customized cell, or a function that returns a cell, or simply raw data.

Reply:
We are passing default styles and default classNames as props from FixedDataTableCell which can be used by client if he don't want to provide any styles to it .And if a client wants to add his own styles he can do that also